src/server/services/helpers/navigation/provide-sub-context-navigation.js

```diff
@@ -0,0 +1,44 @@
+import upperFirst from 'lodash/upperFirst.js'
+import { getEnvironments } from '~/src/server/common/helpers/environments/get-environments.js'
+
+export function provideSubContextNavigation(request, context, h) {
```

Comment:
`context` IMO is the wrong variable name here. As we already have `context` in the `response` which means something different. Can I suggest `section`?